docs/content/latest/api/ysql/commands/ddl_create_table.md

```diff
@@ -85,7 +85,13 @@ Using this qualifier will create a temporary table. Temporary tables are only vi
 
 ### Split Into
 
-The `SPLIT INTO` clause specifies the number of tablets that will be created for the table. This is useful for two data center (2DC) deployments. See example below: [Create CDC table specifying number of tablets](#create-cdc-table-specifying-number-of-tablets).
+The `SPLIT INTO` clause specifies the number of tablets that will be created for the table. This is useful for two data center (2DC) deployments. See example below: [Create CDC table specifying number of tablets](#create-cdc-table-specifying-number-of-tablets). 
```

Comment:
Lets remove the 2DC reference from the description here because it makes it seem like it's just for 2DC which is not the case. Similarly, in the example below too, you don't need to explicitly say 2DC.

Reply:
done